### ISSUE TITLE
SQL execution happening in parallel post update Version 23.2.5.202311191730

### ISSUE THREAD

**Issue by reddykundasrikarun** [dbeaver/dbeaver#21968]
### Description

Previously when I used to execute multiple highlighted queries. Dbeaver would run them sequentially one after the other. However, now its giving a pop-up asking me to accept running them in parallel as below:

<img width="502" alt="image" src="https://github.com/dbeaver/dbeaver/assets/131378773/a6b1a8b2-7344-47ea-a199-bb0f9460e6c9">

Another issue is if I run multiple queries, its always showing statistics tab even when its not needed and unable to disable this one in the settings/preferences as below: 

<img width="809" alt="image" src="https://github.com/dbeaver/dbeaver/assets/131378773/a7f94df7-dd35-4137-8639-ce1f05b3cbe6">

### DBeaver Version

23.2.5.202311191730

### Operating System

windows 11

### Database and driver

Yellowbrick JDBC Driver

**Comment by LonwoLonwo** [comment]
Hello @reddykundasrikarun 

> Previously when I used to execute multiple highlighted queries. Dbeaver would run them sequentially one after the other. However, now its giving a pop-up asking me to accept running them in parallel as below:

So, what is the question here?

**Comment by reddykundasrikarun** [comment]
Hi @LonwoLonwo , 

How to ensure multiple selected queries run in sequence one after another ? 
How to disable the statistics tab as I don't need it?

**Comment by LonwoLonwo** [comment]
> its always showing statistics tab even when its not needed

What queries do you execute? Why statistics is not needed. I need more info here.

**Comment by reddykundasrikarun** [comment]
for 1st scenario of running multiple selected queries, I select all 3 below queries and run them using ctrl+\ in one go so that existing table gets dropped then recreated and then permissions granted.


drop table if exists dbo.temp;
create table dbo.temp ( id integer);
grant select on dbo.temp to public;


Post latest DBeaver update, It always gives a popup as below. If execution happens in parallel then it won't work right 


<img width="429" alt="image" src="https://github.com/dbeaver/dbeaver/assets/131378773/18843e17-99e4-4296-889d-c5811f4cc88b">

.





for 2nd issue scenario. I see there is already another thread :
https://github.com/dbeaver/dbeaver/issues/7704
 
see David Tessier's comment under that on April 3rd. There is an option to disable the "show statistics for multiple queries with results" . However, I cannot see that in my latest DBeaver version as below :

<img width="580" alt="image" src="https://github.com/dbeaver/dbeaver/assets/131378773/351423cf-2dd4-42b6-882f-976715fe6ec1">

**Comment by E1izabeth** [comment]
1. Please try early access version https://dbeaver.io/files/ea/ . We have already fixed it.
2. It's still the same option, just a combo-box instead of checkbox. You can choose `Only when no data option`.
![image](https://github.com/dbeaver/dbeaver/assets/28875055/84d2f31a-6604-4449-b17f-91e9cb0e154e)
